src/test/resources/init-schema.sql

```diff
@@ -0,0 +1,24 @@
+CREATE SEQUENCE hibernate_sequence START WITH 1 INCREMENT BY 1;
+
+CREATE TABLE company (
```

Comment:
init-schema.sql doesn't make sense since it should be created by the hibernate from entities

Reply:
Thanks, removed in https://github.com/Awesome5040/job-search/pull/2/commits/3f306e8fd13302efc31f1ccc30c0c7dc77315b0c